cloudflare-mixin/dashboards/cloudflare-worker-overview.libsonnet

```diff
@@ -0,0 +1,399 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'cloudflare-worker-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local workerCPUTimeQuantilesPanel = {
```

Comment:
I would put the legend on the right since scripts is multiselect. In the case you have multiple scripts selected it would be nice to have the legend on the right. Also since the panel is full width, I believe it wouldn't look bad.